payments/card/after-payment.md

```diff
@@ -593,7 +595,7 @@ Content-Type: application/json
     "payeeId": "{{ page.merchantId }}",
     "isDeleted": true|false,
     "isPayeeToken": false,
-    "cardBrand": "Visa|MasterCard|...",
+    "cardBrand": "Visa | MasterCard | ...",
```

Comment:
No pipes in examples, please. Explain the possible values in below property tables instead.

```suggestion
    "cardBrand": "MasterCard",
```